File: java-suite/src/main/java/de/fraunhofer/iem/classesAndObjects/classes/constructors/multipleConstructors/EmployeeMain.java
package de.fraunhofer.iem.classesAndObjects.classes.constructors.multipleConstructors;

/**
 * Instantiation for the example of multiple constructor
 *
 * @author Ranjith Krishnamurthy
 */
public class EmployeeMain {
    public static void main(String[] args) {
        Employee employee = new Employee("Ranjith", "K", 100);
        System.out.println(employee.firstName);
        System.out.println(employee.lastName);
        employee.age = 200;
        System.out.println(employee.age);
    }
}

File: kotlin-suite/src/main/kotlin/de/fraunhofer/iem/classesAndObjects/classes/constructors/defaultValue/EmployeeMain.kt
package de.fraunhofer.iem.classesAndObjects.classes.constructors.defaultValue

/**
 * Instantiation for the example of default value code.
 *
 * @author Ranjith Krishnamurthy
 */
fun main() {
    val employee: Employee = Employee(age = 100)
    println(employee.firstName)
    println(employee.lastName)
    println(employee.age)
}